### DISCUSSION TITLE
Can the port the SB listens on be changed?  Port Forwarding?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#334] [Q&A]
I'm having a hard time with making sure my remote SB doesn't get bricked.

My setup is Slinger and a remote Slingbox.

I figured out a few things regarding Port Forwarding but have a couple questions remaining:

1. Can the port the SB listens on be changed?  Mine uses 5321 by default.
2. Is there a known list of ISPs or Ports that need to be blocked to prevent bricking?  I simply can't block all WAN traffic because my SB is remote.
3. My SB also auto-generates a UPnP entry in the upstream router, a FiOS G1100.  Can this be disabled from the SB.  I can't disable UPnP on the network, it's not my network.
4. It does look like the G1100 has Parental Controls but I have to use the FiOS app and I don't want to do that.

**Comment by GR0DZ1LLA** [comment]
Some more research:

I was able to write a couple of rules that blocks every single port except 5321.  From the source of the SB.  As opposed to the destination.  Which rule set would be better?  Can I leave the port 5321 open for my Slinger to connect?

![image](https://user-images.githubusercontent.com/115742631/202928821-051eb6bb-ec1f-4319-b660-8ec96d218737.png)

But is this secure?  Does this work to defend against bricking?

**Comment by DoogieHowitzerWMD** [comment]
I have a Fios G1100 at the location of my slingboxes and my server, but when I was initially testing things, I had my server set up in a different location. While testing, I was only able to get the slingboxes to communicate with my server when the ports in the config file were the specific ones that the boxes were using to connect with Sling's servers. I had made new port-forwarding rules for ports I had preferred to use, but I could not get the server to connect using those ports.

That being said, I had not tried taking the external port each of my slingboxes were using, mapping those to only the internal port of the respective slingbox, and remapping the external port on the router to be associated with the specific internal port of the Slingbox. So, for instance, if one of my slingboxes had an external port mapping on the router of 5201, I had not tried associating internal port 5201 for that specific slingbox with a different numbered external port on the router (some high number port in the 5-digit range). It might be possible to connect to your Slinger server with that higher numbered port if you set it up in that fashion.

As far as settings with the G1100 are concerned, I believe if you set up the access control rules to block "any" type of communication at all times specifically to the slingbox, that is a good measure. Then, on first page of the network connections tab where the devices on the network are listed and there is a pulldown of options for the device, you can select "block from internet" for the slingbox. After that and setting the access control, a port forwarding rule that you specify should be the only means the slingbox has to make connect outside of the network. My only concern is whether the UPnP rules would end up allowing the slingbox to sneak around those configurations.
